src/Foundation/Application.php

```diff
@@ -390,12 +390,12 @@ public function setLocale($locale)
     /**
      * Register all of the configured providers.
      *
-     * @var bool $isRetry If true, this is a second attempt without the cached packages.
+     * @param bool $isRetry If true, this is a second attempt without the cached packages.
      * @return void
      */
     public function registerConfiguredProviders($isRetry = false)
     {
-        $providers = Collection::make($this->config['app.providers'])
+        $providers = Collection::make($this->get('config')['app.providers'])
```

Comment:
Because PHPStan is unable to follow through the magic methods, like with the example before with model attributes. This makes it explicit what is happening in a way that PHPStan can follow.